### ISSUE TITLE
Lack of attribution

### ISSUE THREAD

**Issue by rootabeta** [The-fixXxer/AntiPyrus#1]
While I appreciate that imitation is the highest form of flattery, large sections of this program come from my own program, including the entire scanning procedure. While I acknowledge GitHub is a way to distribute and modify code, I would appreciate if the appropriate credit for the work I spent my own free time doing were given, instead of being passed off as your own. If you'd like my email so we can discuss this in more detail, let me know. 

tl;dr, most of this code was originally mine, and I don't appreciate it being passed of as someone else's. In remediation, I would appreciate - at the very least - appropriate crediting of my code, and if that's too much to ask, I ask that this repository be taken down. 

Best,
AntiPirus' original author.

**Comment by DataandGoliath** [comment]
Further update, continued lack of action will be seen as willful failure to comply and may result in legal action.

**Comment by DataandGoliath** [comment]
@The-fixXxer this is your final warning before I begin to seek legal action regarding the unlawful theft of my software. Please reply immediately to inform me that you intend to seek peaceful resolution - failure to do so will tell me that you do not wish to resolve this in a decent manner. Seeing as I've already caught you stealing this code, you might as well resolve this with me in an adult manner.
